native_client/deepspeech.cc

```diff
@@ -478,8 +478,6 @@ DS_CreateModel(const char* aModelPath,
 
   *retval = nullptr;
 
-  DS_PrintVersions();
```

Comment:
@dabinat Removing this will cause that the other clients that uses `deepspeech.cc` directly and not `client.cc` never print the versions, I think printing the versions is useful and not impacting the performance.

Reply:
I will need to change the .NET client if you remove this print, the Python and Node clients has an option to print and exit, I don't see any explicit call to print version from the Java client and maybe @lissyx is relying on this print to see the versions.

Reply:
One can use the -v tag to see the version number, so I’m not sure it’s necessary to print the version every single time it runs, but more importantly, it seems to me like client.cc is a more logical place to put debug info. IMO deepspeech.cc should be pure and only output info relevant to transcription.

Reply:
The version being printed here is to help with our sanity when replying to support requests, as people tend to do all kinds of crazy stuff (including modifying the clients). I agree with you on principle, but in practice, this helps us catch things earlier and not waste time when someone has a problem.